components/PlantDetails/PlantDetails.js

```diff
@@ -75,12 +78,14 @@ export default function PlantDetails({
           <br />
         </StyledPlantNeedsContainer>
         <StyledImageContainer>
-          <Image src={imageUrl} alt={name} width={300} height={300} />
+          <StyledImage src={imageUrl} alt={name} fill={true} />
```

Comment:
Since `fill` can only be `true` or `false` you can skip the `{}` and simply write `fill` which becomes `true`. I you leave it, it's `false`.